templates/paragraphs/paragraph--oe-description-list.html.twig

```diff
@@ -0,0 +1,17 @@
+{#
+/**
+ * @file
+ * Theme override to display the 'Description list' #type paragraph.
+ *
+ * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
+ */
+#}
+
+{% if title is not empty %}
+  <h4 class="fw-bold mb-4">{{ title }}</h4>
+{% endif %}
+
+{{ pattern('description_list', {
+  'items': items,
+  'orientation': orientation,
```

Comment:
shouldn't we send the attributes from here as it is outside the pattern relevant drupal attributes might be passed ?

Reply:
we'll follow up

Reply:
Maybe, but usually it's relevant for paragraphs. We can add it in the future if the need arise.